pkg/lib/inventory/web/client.go

```diff
@@ -264,10 +264,8 @@ func (r *Client) Watch(url string, resource interface{}, h EventHandler) (status
 				pErr = nil
 				return
 			}
+			defer response.Body.Close()
```

Comment:
we can actually move it even higher, just after the check response is not nil (L258)